open-craft/opencraft
# Fix statistics playbook.

This updates the settings to access the Elasticsearch API on the logs server. The Elasticsearch configuration changed recently and we need to update the credentials we use to access it from Ocim when running the `instance_statistics_csv` command.

**Test instructions**:

1. SSH into `stage.ocim.net.opencraft.hosting`.
2. Switch to the `www-data` user, activate the Ocim virtual environment and go to the `/var/www/opencraft` folder (or just attach yourself to the ocim screen session).
3. Run the instance statistics command (example: `make manage -- "instance_statistics_csv --domains=campus-xblocks-test.stage.opencraft.hosting --start-date=2021-06-01 --end-date=2021-06-30"`).
4. Verify that the script completes without any failures (note you might see some N/A in the final result table, but that should be because we don't have much data on stage, not because we weren't able to access elasticsearch).
